airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -67,22 +66,6 @@ public AirbyteConnectionStatus check(final JsonNode config) {
     }
   }
 
-  public static void attemptSQLCreateAndDropTableOperations(final String outputSchema,
-                                                            final JdbcDatabase database,
-                                                            final NamingConventionTransformer namingResolver,
-                                                            final SqlOperations sqlOps)
-      throws Exception {
-    // attempt to get metadata from the database as a cheap way of seeing if we can connect.
-    database.bufferedResultSetQuery(conn -> conn.getMetaData().getCatalogs(), JdbcUtils.getDefaultSourceOperations()::rowToJson);
```

Comment:
if I am reading this correctly, removing this call is the main part of the fix and the rest of the changes are refactoring of attemptSQLCreateAndDropTableOperations method into a separate class. What is the purpose of the refactoring?